mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -2801,7 +2801,7 @@ struct Conv2DRewritePattern : public OpRewritePattern<T> {
 
     auto getGemmC = [&]() {
       // dim of oob ckeck
-      llvm::DenseSet<int> oobCheckDims;
+      llvm::DenseSet<int> oobLoadCheckDims;
```

Comment:
is this truly `oobLoadCheckDims`? this is C matrix, so I assume it shall be `oobStoreCheckDims`?

Reply:
source index, suppoosed it's load?

Reply:
could you elaborate on how do you mean by "source index"?